test_data/manually_generated/empty_v0/manifest.lock

```diff
@@ -0,0 +1,63 @@
+{
+  "lockfile-version": 0,
+  "manifest": {
+    "options": {
+      "systems": [
+        "aarch64-darwin",
+        "aarch64-linux",
+        "x86_64-darwin",
+        "x86_64-linux"
+      ]
+    },
+    "registry": {
+      "defaults": {
+        "subtrees": null
+      },
+      "inputs": {
+        "nixpkgs": {
+          "from": {
+            "owner": "NixOS",
+            "ref": "release-23.11",
+            "repo": "nixpkgs",
+            "type": "github"
+          },
+          "subtrees": [
+            "legacyPackages"
+          ]
+        }
+      },
+      "priority": [
+        "nixpkgs"
+      ]
+    }
+  },
+  "packages": {
+    "aarch64-darwin": {},
+    "aarch64-linux": {},
+    "x86_64-darwin": {},
+    "x86_64-linux": {}
+  },
+  "registry": {
+    "defaults": {
+      "subtrees": null
+    },
+    "inputs": {
+      "nixpkgs": {
+        "from": {
+          "lastModified": 1718132686,
+          "narHash": "sha256-JRinkq+FeAkYnrrK8+Bh+jtLHJBN5jDzSimk1ye00EE=",
+          "owner": "NixOS",
+          "repo": "nixpkgs",
+          "rev": "96b3dae4f8753c1f5ce0d06b57fe250fb5d9b0e0",
+          "type": "github"
+        },
+        "subtrees": [
+          "legacyPackages"
+        ]
+      }
+    },
+    "priority": [
+      "nixpkgs"
+    ]
+  }
+}
```

Comment:
missing final newline

if this is generated with flox we may want to change the writer to add these in the future...

Reply:
Yeah I think that's all of our lockfiles